rosidl_generator_py/test/test_interfaces.py

```diff
@@ -111,6 +111,8 @@ def test_default_values():
     assert 'Bye world' == a.def_string
     assert 'Hello world!' == Strings.DEF_STRING__DEFAULT
     assert 'Hello world!' == a.DEF_STRING__DEFAULT
+    assert "Hello\"world!" == a.DEF_STRING_DELIMITER__DEFAULT
```

Comment:
Why use double quotes here?

Next line it would make sense to use double quotes to avoid the need to escape the single quote.